examples/zephyr/hello/prj.conf

```diff
@@ -0,0 +1,11 @@
+# Application
+CONFIG_GOLIOTH_SAMPLE_COMMON=y
+CONFIG_GOLIOTH_SAMPLE_HARDCODED_CREDENTIALS=y
+
+# Golioth Firmware SDK with all dependencies
+CONFIG_GOLIOTH_FIRMWARE_SDK=y
+
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_STREAM_FLASH=y
+CONFIG_IMG_MANAGER=y
```

Comment:
Why do we need these flash and image symbles for a "hello" sample?

Reply:
It seems like the DFU portions of the SDK are built automatically and I don't see a way to turn them off (the Zephyr SDK has a [CONFIG_GOLIOTH_FW](https://github.com/golioth/golioth-zephyr-sdk/blob/bbae6f8419ed52acd74784ed7e876a0f24e75f8b/net/golioth/Kconfig#L139-L144) symbol for this).

These symbols are necessary to complete the build. But I don't think they should be part of the default config. I've left them here to be removed later when building DFU components is configurable.

Reply:
Works for me! Can you file a ticket to make DFU configurable?